src/CustomerBundle/DependencyInjection/SonataCustomerExtension.php

```diff
@@ -17,14 +17,26 @@
 use Symfony\Component\Config\Definition\Processor;
 use Symfony\Component\Config\FileLocator;
 use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\PrependExtensionInterface;
 use Symfony\Component\DependencyInjection\Loader\XmlFileLoader;
 use Symfony\Component\HttpKernel\DependencyInjection\Extension;
 
 /**
  * @author Thomas Rabaix <thomas.rabaix@sonata-project.org>
  */
-class SonataCustomerExtension extends Extension
+class SonataCustomerExtension extends Extension implements PrependExtensionInterface
 {
+    /**
+     * {@inheritdoc}
+     */
+    public function prepend(ContainerBuilder $container): void
+    {
+        if ($container->hasExtension('twig')) {
+            // add custom form widgets
+            $container->prependExtensionConfig('twig', ['form_themes' => ['@SonataCore/Form/datepicker.html.twig']]);
+        }
```

Comment:
Why is the PR not backwards compatible and why is this necessary now, and was not before?

Reply:
Will look into this

Reply:
I guess it ís backwards compatible (I've rebased). 
Without this, the input used is just a normal text input. Am I missing something or shall i add this fragment to `Sonata\OrderBundle\DependencyInjection\SonataOrderExtension` so the bundles can be used separately?

Reply:
You can do that yeah. Since it is BC, maybe contribute it to the stable branch then?

Reply:
Was a bit confused but had to change the base branch here. (instead of rebasing in git 🤦‍♂)

Reply:
I think you would have to do both things :P